start-finalSetup04ServerProperties

```diff
@@ -89,16 +91,16 @@ function customizeServerProps {
         DIFFICULTY=3
         ;;
       *)
-        echo "DIFFICULTY must be peaceful, easy, normal, or hard."
+        log "DIFFICULTY must be peaceful, easy, normal, or hard."
         exit 1
         ;;
     esac
     setServerProp "difficulty" "$DIFFICULTY"
   fi
 
   if [ -n "$MODE" ]; then
-    echo "Setting mode"
-    MODE_LC=$( echo $MODE | tr '[:upper:]' '[:lower:]' )
+    log "Setting mode"
+    MODE_LC=$( log $MODE | tr '[:upper:]' '[:lower:]' )
```

Comment:
This needs to remain an `echo`

Reply:
Also here, I'm assuming you only want to change line 103?

```suggestion
    MODE_LC=$( echo $MODE | tr '[:upper:]' '[:lower:]' )
```

Reply:
Yes...and using “suggested change” Would have been good of me :)